server/sonar-process/src/main/java/org/sonar/process/LogbackHelper.java

```diff
@@ -52,6 +52,28 @@
  */
 public class LogbackHelper {
 
+  public enum LogProcess {
```

Comment:
check with Java team, but I think classes should start with constants and code rather than embedded types

or maybe it's just a matter of taste to quickly see content of class rather than subtypes

(in this class, all other sub types are at the end)